### ISSUE TITLE
Device already provisioned for current user

### ISSUE THREAD

**Issue by yeswecan** [keybase/client#3686]
Keybase GUI Version: 1.0.16-20160715110019+f0caa34

Keybase forgot me, then I've tried to login again and then I see this:
<img width="541" alt="screen shot 2016-08-01 at 17 17 16" src="https://cloud.githubusercontent.com/assets/1935529/17296886/f509f1be-580b-11e6-9b15-beb7075470d9.png">

**Comment by yeswecan** [comment]
Gonna try to reinstall now.

**Comment by maxtaco** [comment]
Instead of that, can you do `keybsae log send` ?

**Comment by yeswecan** [comment]
Sorry, just trashed it :(. I'll do this instead next time.

**Comment by yeswecan** [comment]
What's interesting is that it didn't require me to re-enter my credentials after I reinstalled it. Is it supposed to be that way? I wasn't able to call `keybase` while it was in Trash.

**Comment by maxtaco** [comment]
Yeah, like the config directory wasn't trashed, nor were your device-specific private keys. You'd have to do a `keybase deprovision` to really blow that stuff away.

**Comment by yeswecan** [comment]
@maxtaco good to know, thanks

**Comment by doudou** [comment]
Got that one right now on 1.0.16-20160726160034+abf37ce (Linux amd64). Funnily enough, it seems that it is only a problem with the GUI. kbfs seems to work just fine (present, can write into it). The keybase command line app also seems to work fine.

**Comment by maxtaco** [comment]
@doudou can you do a `keybase log send`?

**Comment by doudou** [comment]
Just did. I updated the deb and re-ran run_keybase in the meantime, I hope it does not trash the log

log ID: bc0d4b66e46b541ff717811c

Just for my information, is it best practice to run a `log send` anytime I have a bug report "just in case" ?

**Comment by maxtaco** [comment]
The harm in it is that it could reveal some information about the names of your KBFS files.

Thanks @doudou  we will take a look!

**Comment by maxtaco** [comment]
(Cc: @patrickxb  this is CORE-3481 in JIRA)

**Comment by Link-Satonaka** [comment]
Just chiming in: I also had this issue. Deleted my keybase keys and set everything up from scratch through the website, but now I cannot add this device under the name I'd like to, despite me not having any devices associated with my profille.
